discovery-provider/src/api/v1/users.py

```diff
@@ -828,7 +828,28 @@ def get(self, id):
     argument_class=DescriptiveArgument
 )
 users_by_content_node_route_parser.add_argument(
-    "creator_node_endpoint", required=True, type=str
+    "creator_node_endpoint",
+    required=False,
+    type=str,
+    description="DEPRECATED -- use content_node_endpoint",
+)
+users_by_content_node_route_parser.add_argument(
+    "content_node_endpoint",
+    required=False,
+    type=str,
+    description="Get users who have this Content Node endpoint as their primary/secondary",
+)
```

Comment:
woooooo

Reply:
we should add a linear card following this merge to eventually remove the deprecated field. this field is used in a lot of places tho so I suppose that wont happen for a while